### ISSUE TITLE
Option to use game name for series sort of series field is null

### ISSUE THREAD

**Issue by KrivenRaven** [JosefNemec/Playnite#3244]
### Feature description

A toggle which allows a game's name to be used for series sorting rather than sorting all stand-alone games to the end of the list. Allows for near-true series-alpha sort without modifying Sort Name for hundreds of games or other sorting options.

Alternative: allow games without series to be sorted at the _beginning_ of the list instead of the end.

**Comment by mikeymcmikenson** [comment]
Yes! This would help a lot of the people that have asked related sorting questions found on the web.
